data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,325 @@
+			choice
+				`	"Why do you care about these 'flowers'?"`
+					goto care
+				`	"What I search for is not technology."`
+			`	The voice now takes on a calm tone. "Your persistence on this topic is admirable, but allow me to explain my conviction with an example." You say nothing, so the voice continues.`
+			label care
+			`	"Why do humans care about art?" The voice now seems far away yet more present than ever. "Because it amazes you, the same way it amazes us to see another kind grow and overcome their burdens. The lives of species are like a painting that never stop being painted, but when they do stop it is a tragic thing to watch. That is why we try to protect these creations of life - so we can see what heights they reach."`
+			`	The voice pauses before continuing. "I see that you can not be convinced by me. Even though I could simply wipe you out here and now, I do not favor violence. I will let you pass, but I will be watching you as I am-" you can hear the ultimate resolution in the voice, "-the guardian of this space."`
+			`	The voice fades out leaving nothing behind but the feeling of being watched.`
+				decline
+
+mission "Incipias: First Contact I"
+	invisible
+	landing
+	to complete
+		never
+	on enter Dom'us
+		conversation
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. Maybe you can find some more information in nearby systems.`
+				to display
+					not "Incipias: Meet the Quarg: offered"
+			`As you enter the system, you notice ships of a type you have never seen before. Hailing them only gives you static. After some more unsuccessful attempts, you give up. This must be the species the Quarg talked about.`
+				to display
+					has "Incipias: Meet the Quarg: offered"
+
+
+
+mission "Incipias: Meet the Quarg"
+	invisible
+	landing
+	source
+		government "Quarg (Incipias)"
+	to complete
+		never
+	on offer
+		conversation
+			branch "ut divitas" "spec inci"
+				has "flagship planet: Ut Divitas"
+			label "ut divitas"
+			`Your spaceship lands with a soft jolt. A small group of Quarg quickly gathers around your ship, making some simple but clear gestures telling you to wait. After they've realized that you understood, one of them walks away in a hurried manner.`
+			`	Only a moment later the Quarg returns with another Quarg. The new one starts talking with a very strange accent.`
+				goto "both planets"
+			label "spec inci"
+			`	You only wait for a few minutes before three Quarg appear on the landing pad. The one in the middle is wearing clothing that makes clear that it is a military official. The two in the back are wearing full combat gear but do not seem to expect a fight. The one in the middle starts talking in a very strange accent.`
```

Comment:
Probably should make the described waiting periods far longer, closer to the case with the Gegno Quarg finding someone who can speak human. There you go and are moved around the ring a bunch, while here I imagine it would be more the case you just wait for many/a few hours depending on the planet, as the Quarg scramble to get someone available there that can communicate with you.

The reasoning for this is that the Quarg here are far less resourceful than what would be the norm for them, not having a ring at the ready with all the tech there, and whatever teleporty thingies they may have probably wouldn't be as ready to go as in cases where they have a ringworld (assuming this Quarg comes in via some magic Quarg TP machines, which I assume as there just being a Quarg around who happens to speak human in a desolate region by Quarg-standards seemed less likely).

For the Spec Inci one, I'd also remove the mention of the distinctive garb for the Quarg that speaks, as it's a point made throughout the game that the player isn't really ever sure of what Quarg can do what, as seen in Wanderers when you seemingly just happen upon a Quarg who somehow has the authority to send 3 Wardragons to guard the Wanderers, despite the same conversation noting that `"The Quarg do not generally share any information on how their leadership hierarchy works, if such a hierarchy even exists"`.

Reply:
I increased the waiting periods and removed the garb